README.md

```diff
@@ -60,6 +60,10 @@ Traceloop.init(app_name="your_app_name")
 ```
 
 Next, you need to decide where to export the traces to. You can choose between:
+- [x] [Traceloop](#traceloop)
+- [x] [Datadog](#datadog)
+- [x] [Honeycomb](#honeycomb)
+- [x] [SigNoz](#signoz)
```

Comment:
Anything else we want to add here?

Reply:
@tomer-friedman New Relic

Reply:
(I meant btw to remove the explanations from this page and instead link to our docs - https://traceloop.com/docs/python-sdk/exporting